docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_TextEnveloper.htm

```diff
@@ -33,6 +33,8 @@ <h1>Text Enveloper</h1>
 	</tbody>
 </table>
 
+<img src="../../images/mobile_textEnveloper.png" width="15%">
```

Comment:
The image is broken as above and example should be more representative. Please add some values.

Reply:
Fixed